src/web/src/main/java/tupperdate/web/model/chats/Conversation.kt

```diff
@@ -0,0 +1,9 @@
+package tupperdate.web.model.chats
+
+data class Conversation(
+    val id: String,
+    val myId: String,
+    val theirId: String,
+    val myRecipes: List<String>,
+    val theirRecipes: List<String>,
```

Comment:
Use `PictureUrl` here.

Reply:
These are recipe ids, not pictures. Care to explain in more details?